### ISSUE TITLE
[QUESTION] Model performance about context parallel

### ISSUE THREAD

**Issue by cos120** [NVIDIA/Megatron-LM#1340]
Hi, I am using Megatron core 0.9.0 with transformer-engine(1.11+fc03478) and torch 2.4.0 on cuda12.4.

I am doing sft training 5B model with 16K sequence, when I config tp2/pp2 the score of mmlu is 63 and it sames as FSDP, but when I config tp2/pp2/cp2, the score of mmlu is 58, but the training loss is same as tp2/pp2. How can I debug this? Does cp doing something special with sft training?

**Comment by wplf** [comment]
Do you open grad clip?
